indico/modules/rb/schemas.py

```diff
@@ -196,14 +196,14 @@ class Meta:
 
 class ReservationOccurrenceSchema(mm.SQLAlchemyAutoSchema):
     reservation = Nested(ReservationSchema)
-    link = Nested(ReservationOccurrenceLinkSchema)
+    link_id = Number()
```

Comment:
I think you can remove this line altogether, as it's automatically derived from the model (it's an `SQLAlchemyAutoSchema`)

Reply:
Done, and rebased to the latest master branch.